### ISSUE TITLE
Fewer genes detected for CEL-seq2 data

### ISSUE THREAD

**Issue by YOU-k** [BUStools/bustools#55]
Hello,
I followed kallisto_bustools commands in guide you provided.
Here is my code
`kallisto bus -i $work_p/homo_ercc.idx -o nn84/ -x 0,6,14:0,0,6:1,0,0 -t 4 $seq_p/RPI2_S1_R1.fastq.gz $seq_p/RPI2_S1_R2.fastq.gz

bustools correct -w $data_p/barcode_annotation.txt -p output.bus | $bustools sort -t 4 -T ./tmp -p - | $bustools count -o genecount/genes -g $data_p/human_transcript_to_gene.tsv -e matrix.ec -t transcripts.txt --genecounts - 
`
However, there are only hundreds of genes detected for each cell. Do you have idea why would this happen?

**Comment by droplet-lab** [comment]
Hi, I have very similar amount of low counts compared to other pipelines, let me know if you figure this one out! Thanks.

**Comment by wagnerde** [comment]
I have a similar issue when trying to process inDrops V3 libraries with the kb python wrapper.  The end result is hundreds of UMI counts per cell, whereas other pipelines analyzing the same dataset yield thousands of UMIs per cell.

**Comment by droplet-lab** [comment]
@wagnerde Yes I've actually noticed this problem for indrops v3 libraries as well, but I would guess the library type is quite similar to Cel-seq 2. The problem has been there since the release of bustools, but am glad to see I am not the only one that is having this problem. It would be great if this problem could be fixed, but unfortunately this seems to not be an immediate priority.
